### ISSUE TITLE
Read only?

### ISSUE THREAD

**Issue by DirkSimons** [mrsauravsahu/vscode-manager#51]
On first install (vscode  1.65.2 on m1 mac), made a new profile and tried to change the name.  Didn't work.  I opened the profile.json through a right click on the profile, but can't edit the file: 'Cannot edit in read only Editor'

**Comment by mrsauravsahu** [comment]
Yes, currently, the profile.json can be modified only through UI.

Can you explain the error when you tried to rename the profile? (I'm assuming you tried 'Rename' through the sidebar)

Can you try the refresh button after you renamed the profile?

![Screenshot from 2022-03-27 00-47-13](https://user-images.githubusercontent.com/9134050/160253986-ec4deb3b-472c-4ffc-8870-63cdc1ce1894.png)

**Comment by jrep41** [comment]
Hello. Excellent work. To me the same thing happens to me. The profile can be renamed on the side but when launching it, the title bar keeps the old name and won't let me change it in the editor. 
![2022-04-02 (2)](https://user-images.githubusercontent.com/11485536/161400031-18583555-05fa-4290-b59d-2c6c5e1a6d2f.png)

**Comment by mrsauravsahu** [comment]
Ah, got it. Yes, I'd never implemented this. Should be pretty simple to do. Will add this to the next version.
